docs/pages/example/style-ocean-depth-data.html

```diff
@@ -0,0 +1,64 @@
+<div id='map'></div>
+
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/streets-v10',
+    center: [142.591669, 11.373335], // coordinate position
+    zoom: 3, // starting zoom
+    minZoom: 3, // minimum zoom
+    maxZoom: 5, // maximum zoom
```

Comment:
I don't think we need these comments on lines 7-10 and lines 15 and 21. But can you add one or two explanatory comments that briefly explain how the expression is working?